NEWS

```diff
@@ -45,6 +45,12 @@ CHANGES WITH 255 in spe:
         * The NAMING_BRIDGE_MULTIFUNCTION_SLOT naming scheme has been deprecated
           and is now disabled.
 
+        * SuspendMode=, HibernateState= and HybridSleepState= in the [Sleep]
+          section of systemd-sleep.conf are now deprecated and have no effect.
+          They did not (and could not) take any value other than 'sleep' for the
```

Comment:
/sys/power/disk is only used for hibernation. `suspend` doesn't use this file.

Reply:
fixed